stock_prices/stock_prices.py

```diff
@@ -1,15 +1,41 @@
 #!/usr/bin/python
 
 import argparse
+# import math
+#find max profit from list of stock prices
+#evaluate high and low to find max difference
+#sell at high number
+testing_price_list = [78, 2, 3, 0, 75, 86, 107]
 
-def find_max_profit(prices):
-  pass
+def find_max_profit(prices): #receives as input a list of stock prices
+  #use iterative sorting
+  max_profit = prices[1] - prices[0]
```

Comment:
Nice job here. We can set the first two elements as our calculation for max profit and then iterate through and update when necessary.